alisa_wrap.go

```diff
@@ -23,18 +23,27 @@ const (
 	readResultsBatch = 20
 )
 
-func (ali *alisa) exec(cmd string) (*alisaTaskResult, error) {
+func (ali *alisa) exec(cmd string) error {
```

Comment:
Maybe we can export `Exec` so that submit task by:
``` golang
ali := NewAlisa()
ali.Exec(paiCmd, verbose=true)
```

Reply:
Shouldn't this `exec` function be wrapped under `db.Exec`?